### PR TITLE
ED-2487 update header/footer

### PR DESCRIPTION
**Pull request by eadpearce**
Header/footer now updated with dev urls 
https://uktrade.atlassian.net/browse/ED-2487

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -117,8 +117,20 @@ DEBUG_SET_ENV_VARS := \
 	export COMPANIES_HOUSE_CLIENT_SECRET=debug-client-secret; \
 	export SECURE_HSTS_SECONDS=0; \
 	export PYTHONWARNINGS=all; \
-	export PYTHONDEBUG=true
-
+	export PYTHONDEBUG=true; \
+	export EXPORTING_NEW=/new; \
+	export EXPORTING_OCCASIONAL=/occasional; \
```
Comment (konradko) [inline makefile]: All these env vars also need adding to https://github.com/uktrade/directory-ui-export-readiness/pull/51/files#diff-cd5dbf3629c558b0104ba8f0937d6816L36 with a prefix `DIRECTORY_UI_EXPORT_READINESS_` - annoying, but needed for tests

File: ui/settings.py [hunk 2]
```diff
@@ -216,6 +217,27 @@
 SECURE_HSTS_SECONDS = int(os.getenv('SECURE_HSTS_SECONDS', '16070400'))
 SECURE_HSTS_INCLUDE_SUBDOMAINS = True
 
+# HEADER/FOOTER URLS
+# EXPORTING PERSONAS
+EXPORTING_NEW = os.environ["EXPORTING_NEW"]
```
Comment (konradko) [inline ui/settings.py]: We should default to prod links on all these

Comment (eadpearce) [reply ui/settings.py]: the header/footer context processor does that if the link isn't specified. they're listed in directory-constants https://github.com/uktrade/directory-constants/blob/master/directory_constants/constants/urls.py

File: makefile (hunk 1 above)
Comment (eadpearce) [reply makefile]: okay. will fix

File: ui/settings.py (hunk 2 above)
Comment (eadpearce) [reply ui/settings.py]: okay this was also breaking the tests. have fixed now

File: ui/settings.py [hunk 3]
```diff
@@ -11,6 +11,8 @@
 """
 
 import os
+# import all production urls from directory_constants
+from directory_constants.constants import urls
```
Comment (richtier) [inline ui/settings.py]: we can avoid the above comment by giving the variable a explanatory name:

```
from directory_constants.constants import urls as default_urls
```

**Pull request merged**